tools/stats/monitor.py

```diff
@@ -1,77 +1,54 @@
 #!/usr/bin/env python3
 import datetime
-import importlib.util
 import json
-import subprocess
-import sys
+import signal
 import time
 from typing import Any, Dict, List
 
+import psutil  # type: ignore[import]
+import pynvml  # type: ignore[import]
```

Comment:
Probably no longer needed `type: ignore[import]`?

Reply:
i think they are still needed for mypy